stable/mariadb/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: mariadb
-version: 0.6.3
+version: 0.6.4
```

Comment:
You shouldn't be bumping the chart version of the stable/mariadb chart in this PR. I believe you did not intend to do so.